packages/cli/src/MCP.ts

```diff
@@ -0,0 +1,183 @@
+import { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js'
+import { StdioServerTransport } from '@modelcontextprotocol/sdk/server/stdio.js'
+import type { PrismaConfigInternal } from '@prisma/config'
+import { Command, link } from '@prisma/internals'
+import { spawn } from 'child_process'
+import { z } from 'zod'
+
+import { createHelp } from './platform/_lib/help'
+
+function spawnAsPromise(
+  command: string,
+  args: string[] = [],
+  cwd: string,
+): Promise<{ stdout: string; stderr: string }> {
+  return new Promise((resolve) => {
+    const child = spawn(command, args, { cwd: cwd || process.cwd() })
+
+    let stdout = ''
+    let stderr = ''
+
+    child.stdout.on('data', (data) => {
+      stdout += data
+    })
+
+    child.stderr.on('data', (data) => {
+      stderr += data
+    })
+    child.on('close', () => {
+      resolve({ stdout, stderr })
+    })
+
+    child.on('error', (err) => {
+      resolve({ stdout: '', stderr: err.message })
+    })
+  })
+}
+
+export class Mcp implements Command {
+  public static new(): Mcp {
+    return new Mcp()
+  }
+
+  private constructor() {}
+
+  public help = createHelp({
+    options: [['--early-access', '', 'Enable early access features']],
+    examples: ['prisma mcp --early-access'],
+    additionalContent: [
+      'Starts an MCP server to use with AI development tools such as Cursor, Windsurf and Claude Desktop',
+      `For additional help visit ${link('https://pris.ly/cli/mcp')}`,
+    ],
+  })
+
+  public async parse(_argv: string[], _config: PrismaConfigInternal): Promise<string | Error> {
+    const server = new McpServer({
+      name: 'Prisma',
+      version: '1.0.0',
+    })
+
+    server.tool(
+      'migrate-status',
```

Comment:
Should these be prefixed with `prisma-` or the tools from different servers are inherently namespaced already?